docs/.vuepress/theme/Layout.vue

```diff
@@ -76,7 +76,8 @@ export default {
   },
   computed: {
     isLandingPage() {
-      return this.$page.frontmatter && this.$page.frontmatter.layout === 'home'
+      const layouts = ['home', 'landingpage']
```

Comment:
Do we want this? Why not just keep as a standard `page` class? See my other comments w/ images on why I think we should keep the padding the same as other pages.

Reply:
The `page` class is very limiting because it's tailored to long-form content rather than rich landing pages. For example, setting full-width sections (e.g. to set a background color) is a huge hassle. `landingpage` removes those limitations, while avoiding pulling in the hero (which should be called in the homepage markdown anyway).

Reply:
Got it. In that case, if we plan to use this across these new landing pages, I recommend we create a new component for this custom layout. For instance, we could create a new `<LandingPage />` component & in these markdown files specify `layout: LandingPage` in the frontmatter. Then we apply specific classes / styling within that LandingPage.vue component.

More on how to accomplish that here:
https://vuepress.vuejs.org/theme/default-theme-config.html#custom-page-class